pkg/volume/csi/csi_block.go

```diff
@@ -189,7 +189,7 @@ func (m *csiBlockMapper) stageVolumeForBlock(
 		nil /* MountOptions */)
 
 	if err != nil {
-		return "", errors.New(log("blockMapper.stageVolumeForBlock failed: %v", err))
```

Comment:
nit: if we want to keep these error messages for consistency, then we can perhaps wrap with `%w` and then use `error.Is()` check?

Reply:
I like `%w`, however, I don't like a project where some errors are wrapped with `%w` and some are not.

Perhaps we could rework whole uncertain state (filesystem & block) to `%w` in a subsequent PR.

Reply:
alright lets do this in a follow up - https://github.com/kubernetes/enhancements/issues/1589